net/WebSocketHandler.hpp

```diff
@@ -60,17 +60,21 @@ class WebSocketHandler : public ProtocolHandlerInterface
     /// isClient: the instance should behave like a client (true) or like a server (false)
     ///           (from websocket perspective)
     /// isMasking: a client should mask (true) or not (false) outgoing frames
-    WebSocketHandler(bool isClient, bool isMasking) :
+    WebSocketHandler(bool isClient, bool isMasking)
+        :
 #if !MOBILEAPP
-        _lastPingSentTime(std::chrono::steady_clock::now()),
-        _pingTimeUs(0),
-        _isMasking(isClient && isMasking),
-        _inFragmentBlock(false),
-        _key(isClient ? PublicComputeAccept::generateKey() : std::string()),
+        _lastPingSentTime(std::chrono::steady_clock::now() -
```

Comment:
This isn't relevant for `isClient=true` case, since only the server (i.e. `isClient=false`) sends pings.